convert2rhel/unit_tests/actions/system_checks/convert2rhel_latest_test.py

```diff
@@ -43,7 +43,7 @@ def convert2rhel_latest_version_test(monkeypatch, tmpdir, request, global_system
     marker = request.param
     monkeypatch.setattr(convert2rhel_latest, "installed_convert2rhel_version", marker["local_version"])
 
-    run_subprocess_mocked = mock.Mock(spec=utils.run_subprocess, return_value=(marker["package_version"], 0))
+    run_subprocess_mocked = unit_tests.RunSubprocessMocked(return_string=marker["package_version"])
```

Comment:
Commenting here since it's the only line changed. 

`from six.moves import mock` in line 30 is not being used anymore.

Reply:
Fixed, thanks